modules/addTask.js

```diff
@@ -1,6 +1,7 @@
 import { getTaskFromLocalStorage, setTaskIntoLocalStorage } from './localStorage.js';
 
 const addTask = (text) => {
+  console.log('addTask called');
```

Comment:
- [ ] [OPTIONAL] Kind remainder let's make sure we don't have a console.log in our production